src/util/ParallelMultiwayMerge.h

```diff
@@ -78,6 +78,7 @@ CPP_template(typename T, bool moveElements, typename SizeGetter,
       it2_{};
 
  public:
+  using value_type = std::vector<T>;
```

Comment:
The much better fix would be to make the `AsyncStream` correct, in that it doesn't use
`T::value_type` but `ql::ranges::range_value<T>`, then other people also won't stumble upon this anymore.